### PR TITLE
fix #37159 unn.com.ua

### LINKED ISSUE #37159: unn.com.ua
### Issue URL (Incorrect Blocking)
[https://www.unn.com.ua/ru/news/1814125-pidpal-studiyi-anime-u-yaponiyi-stav-vidomim-ymovirniy-motiv-zlochintsya-yakiy-vbiv-33-osobi](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.unn.com.ua%2Fru%2Fnews%2F1814125-pidpal-studiyi-anime-u-yaponiyi-stav-vidomim-ymovirniy-motiv-zlochintsya-yakiy-vbiv-33-osobi)
### Comment
Username: @spok1
> .fancybox-overlay
> .fancybox-wrap
> ломают отображение

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Stealth mode options: | Hide your search queries, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/37159

### CODE REVIEW

File: SocialFilter/sections/popups.txt [hunk 1]
```diff
@@ -8,6 +8,10 @@
 ||mailget.net^$third-party
 ||my.hellobar.com^$third-party
 !
+!+ PLATFORM(android, ios, ext_android_cb)
+unn.com.ua##.fancybox-overlay
+!+ PLATFORM(android, ios, ext_android_cb)
+unn.com.ua##.fancybox-wrap
```
Comment (BlazDT) [inline SocialFilter/sections/popups.txt]: Same issue as reported for those devices then.

Please try these rules:

##body > .fancybox-opened
##body > .fancybox-opened + .fancybox-overlay

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/popups.txt (hunk 1 above)
Comment (zloyden) [reply SocialFilter/sections/popups.txt]: `##body > .fancybox-opened + .fancybox-overlay` hide overlay, but not a popup.

`unn.com.ua##body > .fancybox-opened`
`unn.com.ua##body > .fancybox-overlay` these are ok, but i get the same problem (now in mobile version) - can't open photo from the article in issue.

Comment (BlazDT) [reply SocialFilter/sections/popups.txt]: The first is hiding the popup for me, 2nd one overlay. But not the photo on mobile for me.

Else please set the cookie which gets created when closing the initial popup on mobile.

Comment (zloyden) [reply SocialFilter/sections/popups.txt]: Please check this rules on your device. It seems ok. Without `#%#` rule I noticed a popup for 1 second.

`unn.com.ua##body > .fancybox-opened`
`unn.com.ua##body > .fancybox-opened + .fancybox-overlay`
`unn.com.ua#%#document.cookie = "scps=1";`

Comment (BlazDT) [reply SocialFilter/sections/popups.txt]: As mentioned first and 2nd rule are working fine for me without the need of cookie rule.
If they are malfunctioning for you, cookie is enough for android, while ios and cb do not support cookie rule.

File: SocialFilter/sections/popups.txt [hunk 2]
```diff
@@ -9,9 +9,9 @@
 ||my.hellobar.com^$third-party
 !
 !+ PLATFORM(android, ios, ext_android_cb)
```
Comment (BlazDT) [inline SocialFilter/sections/popups.txt]: Android does not need the rules when you have cookie rule (applies to Annoyances too).

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**